CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
@maximedenes: I don't know what to advise. Listing bug fixes was originally an idea from you if I remember correctly: it started for 8.5pl1 with commits from you.

For pl versions, it seems to make sense because bug fixes is what distinguishes a plX version from the previous version.

But when a list of bugs starts to contain 400 items or even a somehow arbitrary excerpt of 400 items, that we don't really have the manpower to give an appropriate description of the change for each of the bug, that no one can reasonably read all 400 items, that such a list is taking more space in the description of the changes that the structured part of the file, I start to be worried.

Maybe a discipline from each of us to associate to each fix a relevant comment in the CHANGES file at the time the fix is committed can be imagined (so that the task of reporting bug fixes is not titanic at the time of a release but incremental). Then, the change could be inserted dynamically at the right place (e.g. at the line describing "regular" subst, one may add the comment "#3070 and #5149 are also fixed"; for #3035, one could add an item in the Vernacular commands paragraph that Arguments diallow trailing arguments, mentioning #3045 in parentheses; a paragraph Efficiency could be added to host the changes reported for #3441, #3450; #3957, #4464 and many others could be simply dropped because uninformative enough to be useful, etc.).

But, yes, if we decide that this is worth the effort, it seems that, as you suggest by proposing a cep, the effort can only be a collective one.